db/install.xml

```diff
@@ -60,6 +60,7 @@
         <FIELD NAME="deleted_at" TYPE="int" LENGTH="12" NOTNULL="false" SEQUENCE="false"/>
         <FIELD NAME="enable_stream_url" TYPE="int" LENGTH="1" NOTNULL="true" SEQUENCE="false"/>
         <FIELD NAME="enable_download_url" TYPE="int" LENGTH="1" NOTNULL="true" SEQUENCE="false"/>
+        <FIELD NAME="registration_flag" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE = "false"/>
```

Comment:
@Ash-5 just for consistency this could be `enable_regisration`. Sorry for being too picky and choosy.

Reply:
@meyyappanv better I fix this now and I have fixed and tested.